ChatDemo-UI3.0/ChatDemo-UI3.0/Code/Controllers/Contacts/EMContactInfoViewController.m

```diff
@@ -201,16 +201,17 @@ - (UIView *)tableView:(UITableView *)tableView viewForHeaderInSection:(NSInteger
 - (void)actionSheet:(UIActionSheet *)actionSheet clickedButtonAtIndex:(NSInteger)buttonIndex
 {
     if (buttonIndex != actionSheet.cancelButtonIndex) {
+        
         WEAK_SELF
         [MBProgressHUD showHUDAddedTo:self.view animated:YES];
-        [[EMClient sharedClient].contactManager deleteContact:_model.hyphenateId completion:^(NSString *aUsername, EMError *aError) {
-            [MBProgressHUD hideAllHUDsForView:weakSelf.view animated:YES];
```

Comment:
the line number changed from 207 to 206 for [MBProgressHUD hideAllHUDsForView:weakSelf.view animated:YES];